zerver/lib/rest.py

```diff
@@ -57,7 +57,7 @@ def rest_dispatch(request, **kwargs):
     # Override requested method if magic method=??? parameter exists
     method_to_use = request.method
     if request.POST and 'method' in request.POST:
-        method_to_use = request.POST['method']
+        method_to_use = cast(Text, request.POST['method'])
```

Comment:
Is the purpose of these `cast` calls to assert these aren't None, or something else?  For the latter, `assert method_to_use is not None` might be clearer.